document-message/document-message.yml

```diff
@@ -0,0 +1,5 @@
+tagline: Transfer data to another application
+desc: The document message reliably transfers data from one application to another.
+helps: Ballerina supports a wide range of libraries for handling various messaging protocols, including REST, GraphQL, gRPC, and more. Additionally, it is equipped to handle a diverse array of MIME type attachments.
+tags: ["Document Message", "File Transfer", "Message Channnel", "Message Endpoint"]
```

Comment:
[File Transfer](https://www.enterpriseintegrationpatterns.com/patterns/messaging/FileTransferIntegration.html) is not an EIP.

Reply:
In last review it was decided to remove non-eip tags.